app/controllers/api/api_docs_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class Api::ApiDocsController < Api::BaseController
+  before_action :find_service
+  before_action :find_service_apis_collection, only: :new
```

Comment:
IMO `before_action` is overkill in this cause, better to just call `service_apis_collection` in the `new` method.

Reply:
in my opinion having a method that returns collection is much better then using instance variable that is being created in a `before_action`. it's much more readable when controller gets fat.

Reply:
> having a method that returns collection is much better then using instance variable that is being created in a before_action

When the instance variable has to be created is clearly a subject to debate, ok, but the reason it needs to be created is for the view. And I don't have time to refactor that now (which is done the same way for the other controller and view and I am reusing that view). So the variable must be created either in a `before_action` like now or inside the `new` without any additional method.

Reply:
Both has reason, neither is explicit enough to say: "it is only used in the view."

I think with a presenter then all is settled, as you define the method in the presenter and use it in the view ... no need of before_action

Reply:
Yeah, too much only for this 😂